packages/notification/src/model/notificationQueue.ts

```diff
@@ -18,6 +18,7 @@ const errorSchema = {
 const notificationQueueSchema = new Schema({
   subject: { type: String, required: true },
   body: { type: String, required: true },
+  recipientEmail: { type: String, required: true },
```

Comment:
Let us suppose there were some notifications pending on the `NotificationQueue` before we deploy this change and after deploying we will be "assuming" `recipientEmail` is available in the previous entries as well